src/AlertBox/AlertBox.jsx

```diff
@@ -0,0 +1,66 @@
+import _ from "lodash";
+import classnames from "classnames";
+import React, {PureComponent, PropTypes} from "react";
+
+import "./AlertBox.less";
+import "./AlertBoxIcons.less";
+
+const closeIcon = <svg viewBox="0 0 1792 1792" xmlns="http://www.w3.org/2000/svg"><path d="M1490 1322q0 40-28 68l-136 136q-28 28-68 28t-68-28l-294-294-294 294q-28 28-68 28t-68-28l-136-136q-28-28-28-68t28-68l294-294-294-294q-28-28-28-68t28-68l136-136q28-28 68-28t68 28l294 294 294-294q28-28 68-28t68 28l136 136q28 28 28 68t-28 68l-294 294 294 294q28 28 28 68z" /></svg>;
```

Comment:
Can you make this a function that returns this and then use it as a component? Like:

```
function CloseIcon() {
 return <svg>...</svg>
};

//...
render() {
  return <CloseIcon />...
}
```

In the current case, all `AlertBox`'es on the screen will use the same instance of the component if you have multiple of them onscreen, I'm not so sure how React would deal with that especially if the Javascript enacts any modification to the SVG styling / elements via js. more robust to just create a new instance via the above code.